### DISCUSSION TITLE
How to create browser or window with Chrome toolbar when ChromeRuntime is enabled?

### DISCUSSION THREAD

**Discussion by n303ef** [cefsharp/CefSharp#4780] [Q&A]
I would like to enable Chrome toolbar (downloads, settings, etc). I noticed that a window with it can be created if I right-click on some link and then press "Open in new window". But, first, I would like to do it myself with any desirable URL, and, second, this new window doesn't work - it always shows "Oops" error.

**Comment by amaitland** [comment]
Not currently supported.

The `Chrome` window cannot be embedded as a child e.g. `WinForms` or `WPF`. It has to be a standalone window, `CefSharp` really wasn't designed for this. It's possible, though would take a lot of work.

Have you considered `puppeteer-sharp` with `Chrome` in a `non-headless` state?

**Comment by n303ef** [comment]
I found a way to do it. We just need to open new window with JavaScript:

`browser.EvaluateScriptAsync("window.open('" + url + "')");`

We can also access it's IBrowser instance through ILifeSpanHandler's OnAfterCreated event.
